source/api/passwords.md

```diff
@@ -166,4 +166,12 @@ Accounts.emailTemplates.resetPassword.from = function () {
    // Overrides value set in Accounts.emailTemplates.from when resetting passwords
    return "AwesomeSite Password Reset <no-reply@example.com>";
 };
+Accounts.emailTemplates.verifyEmail = {
+   subject() {
+      return "Activate your account now!";
+   },
+   text (user, url) {
+      return "Hey " + user + "! You might not have access to all our features by clicking this email validation link: " + url;
+   }
+};
```

Comment:
Having this example is a great idea.  Can you just remove the space after `text` and before the `(` and change the text to something less ominous, like:

    To gain access to all our features, click this link to verify your e-mail:

😃